test/integration/default/serverspec/default_spec.rb

```diff
@@ -0,0 +1,3 @@
+require 'serverspec'
+
+set :backend, :exec
```

Comment:
Remove default_spec, or server_spec?  I think you meant server_spec but let me know.